cpu/sam0_common/include/periph_cpu_common.h

```diff
@@ -877,11 +877,7 @@ static inline bool cpu_woke_from_backup(void)
  * @brief ADC Channel Configuration
  */
 typedef struct {
-    union {
-        uint32_t inputctrl; /**< ADC channel pin multiplexer value  */
-        uint32_t muxpos;    /**< ADC channel pin multiplexer value
-                                 @deprecated, use inputctrl instead */
-    };
+    uint32_t inputctrl; /**< ADC channel pin multiplexer value  */
```

Comment:
```suggestion
    uint32_t inputctrl;     /**< ADC channel pin multiplexer value  */
```

Just for alignment purposes, if I counted correctly.

Reply:
Adjusted, I force pushed directly :')

Reply:
ping @mguetschow

Reply:
ping @mguetschow

Reply:
whoopsie :eyes: